hack/contributors.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+
+from github import (Github, GithubException)
+
+import os
+import requests
+import sys
+
+main_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+
+def github_login():
+    token = os.getenv('GITHUB_TOKEN')
+    if not token:
+        token_file = os.path.join(main_path, '.gh-token')
+        if not os.path.exists(token_file):
+            print('GITHUB_TOKEN not set, and {} not found.'.format(token_file))
+            sys.exit(1)
+        token = open(token_file).read().strip()
+    return Github(token)
+
+def get_contributions_from_gh():
+    repos = [
+        '.github',
+        'community',
+        'flux2',
+        'flux2-kustomize-helm-example',
+        'flux2-multi-tenancy',
+        'go-git-providers',
+        'helm-controller',
+        'image-automation-controller',
+        'image-reflector-controller',
+        'kustomize-controller',
+        'notification-controller',
+        'pkg',
+        'source-controller',
+        'source-watcher',
+        'terraform-provider-flux',
+        'website',
+        'webui'
```

Comment:
Are these all the repos we want to check - or should we include *all* /fluxcd repos?